src/components/ebay-star-rating-select/examples/02-fieldset/template.marko

```diff
@@ -0,0 +1,12 @@
+class {}
+
+<fieldset>
+    <legend>Select rating</legend>
```

Comment:
A small thing, but legend and aria-label strings are different. Let's consolidate on "Rate product" for both